lib/rubocop/cop/style/symbol_array.rb

```diff
@@ -39,7 +39,7 @@ class SymbolArray < Base
         minimum_target_ruby_version 2.0
 
         PERCENT_MSG = 'Use `%i` or `%I` for an array of symbols.'
-        ARRAY_MSG = 'Use `%<prefer>s` for an array of symbols.'
+        ARRAY_MSG = 'Use `[...]` for an array of symbols.'
```

Comment:
Can you tweak the message?

```suggestion
        ARRAY_MSG = 'Use array brackets `[...]` for an array of symbols.'
```

Reply:
Sure, thank you for pointing that out. I too was afraid that this `[...]` might be a bit confusing.